docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-product-option-feature-integration.md

```diff
@@ -0,0 +1,333 @@
+---
+title: Marketplace Product Option feature integration
+last_updated: Jul 28, 2021
+Description: This document describes the process how to integrate the Marketplace Product Option feature into a Spryker project.
+template: feature-integration-guide-template
+---
+
+This document describes how to integrate the Marketplace Product Option feature into a Spryker project.
+
+
+## Install feature core
+
+Follow the steps below to install the Marketplace Product Option feature core.
+
+### Prerequisites
+
+To start feature integration, integrate the required features:
+
+| NAME | VERSION | LINK |
+| --------------- | ------- | ---------- |
+| Spryker Core         | {{page.version}}      | [Spryker Core Feature Integration](https://documentation.spryker.com/docs/spryker-core-feature-integration) |
+| Marketplace Merchant | {{page.version}}      | [Marketplace Merchant feature integration](/docs/marketplace/dev/feature-integration-guides/{{page.version}}/marketplace-merchant-feature-integration.html) |
```

Comment:
Product Option feature is missing.

Reply:
added the feature, but related IG is absent in core documentation